static/src/javascripts/projects/common/modules/article/space-filler.ts

```diff
@@ -34,6 +34,12 @@ class SpaceFiller {
 				})
 				.catch((ex) => {
 					if (ex instanceof SpaceError) {
+						if (options?.debug) {
+							const event = new CustomEvent(
+								'spacefiller-complete',
+							);
+							document.dispatchEvent(event);
+						}
```

Comment:
Minor - could this be extracted to a function which can be used in both places where we dispatch this event.